package.json

```diff
@@ -35,6 +35,7 @@
     "@fortawesome/react-fontawesome": "^0.1.14",
     "@tinymce/tinymce-react": "~3.8.4",
     "availity-reactstrap-validation": "^2.3.0",
+    "aws-iot-device-sdk": "^2.2.11",
```

Comment:
i thought we only needed it for `api/`? to undo this do

```
npm uninstall aws-iot-device-sdk --save
```

Reply:
Okay so after uninstalling from `Core-v4/package.json` api-testing doesn't work, so I think we need it in both

Reply:
sounds good, resolving